doc/source/catalog_generation.rst

```diff
@@ -436,6 +530,241 @@ good and the other contaminated.
 Should the catalogs fail this test, neither type of catalogs will be written out to disk for this visit.
 
 
-Segment Photometric Catalog Generation
-=======================================
-Michele's documentation goes here!
+4: Segmentation Catalog Generation
+----------------------------------
+
+4.1: Source Identification with PhotUtils
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+For the segmentation algorithm the
+`photutils.segmentation <https://photutils.readthedocs.io/en/stable/segmentation.html>`_ Astropy 
+tool is used to identify sources in the background-subtracted multi-filter detection image. 
+To identify a signal as a source, the signal must have a minimum number
```

Comment:
This would also be a good point to add a sentence clarifying that the background used here is the common one described in the background-computation section above in order to tie those results into both catalog generation steps.

Reply:
Ditto and Done.